### DISCUSSION TITLE
Default styling for Document

### DISCUSSION THREAD

**Discussion by tannerPriest** [dolanmiu/docx#1297] [Q&A] [answered]
Hello, sorry but new to Docx generation, I cant seem to find documentation on how to set the default styles for a document. 

My use case is that I want all font generated to be 'bold' and have the font-family of 'Calibri'

This is what Ive tried so far: 

```js
const paragraphPhoto = new Paragraph({
            text,
            style: `font-family: "Calibri; bold: true;`,
        })

   const doc = new Document({
        sections: [{
            properties: {},
            children: [paragraphPhoto]
        }],
    });
```
I cant seem to get it too work. There are no options for Document.styles.Paragraph, only listParagraph. I can not use Text Runs, because I am using a template generator to render images and the generator does not like the template variable in a TextRun. Said it must be a blank paragraph.

**Comment by JsProgrammer12** [comment, marked as answer]
Try something like this....

```
const paragraphPhoto = new Paragraph({
    text,
    style: 'myAwesomeStyle',
})

const doc = new Document({
    styles: {
        paragraphStyles: [{
             id: 'myAwesomeStyle',
             run: {
                 font: 'Calibri',
                 bold: true
             }
        }]
     },
     sections: [{
            properties: {},
            children: [paragraphPhoto]
     }],
});
```

**Reply by tannerPriest** [reply]
Woah that works. Ignore me, I was confused about the id because I was thinking it was looking for the id of the element, and that it was actually the id for the style. 

Im going to go walk into a door now. 

Thank you it works great!
